es6-shim.js

```diff
@@ -1954,11 +1965,13 @@
     sinh: function sinh(value) {
       var x = Number(value);
       if (!globalIsFinite(x) || x === 0) { return x; }
-
-      if (_abs(x) < 1) {
-        return (Math.expm1(x) - Math.expm1(-x)) / 2;
+      var s = _sign(x);
+      var a = _abs(x);
+      if (a < Math.log1p(2 / Number.EPSILON - 1) / 2) {
+        var t = Math.expm1(a);
+        return s * (t + t / (t + 1)) / 2;
       }
-      return (_exp(x - 1) - _exp(-x - 1)) * Math.E / 2;
+      return s * (Math.expm1(a - 1) + 1) * (Math.E / 2);
```

Comment:
linking to discussion here - https://github.com/paulmillr/es6-shim/pull/338#discussion_r28573486 - we may want to add a comment or a test enforcing that these parens are here.